styles.css

```diff
@@ -277,6 +274,25 @@ a {
     }
 }
 
+.card { 

```

Comment:
I'm not 100% sure this has been implemented correctly throughout (especially over screen size breakpoints) but the idea is that we've created a 'card' class that has all the styling for HTML elements that act as little info cards on the site i.e. both team members and services. Below we have a bunch of extra styling for 'blue cards' which is what was services. 

The advantage of this is that any styling you want to apply to 'cards' will likely apply to similar components as you expand the website.